tests/test_rocket.py

```diff
@@ -1,9 +1,31 @@
 import unittest
 
+from rocket.rocket import Rocket
 
-class RocketTestCase(unittest.TestCase):
-    def test_something(self):
-        self.assertEqual(True, True)
+
+def test_build_python(python_rocket):
+    python_rocket._build()
+    assert python_rocket.wheel_file
+    assert python_rocket.wheel_path
+
+
+def test_build_poetry(poetry_rocket):
+    poetry_rocket._build()
+    assert poetry_rocket.wheel_file
+    assert poetry_rocket.wheel_path
+
+
+def test_build_wrong_project_location():
```

Comment:
Could you please add a short sentence with what we are testing here (and all other tests, in english). On the long run tests without an uncontestable purpose tend to frustrate people and get removed :).

Reply:
👍